src/main/java/org/neo4j/gis/spatial/DefaultLayer.java

```diff
@@ -19,10 +19,7 @@
  */
 package org.neo4j.gis.spatial;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
```

Comment:
One thought: Perhaps IntelliJ did that? You should check it, and perhaps find out how you can disable automatic wildcard imports.